core/api/rpc/wsc.cpp

```diff
@@ -87,27 +88,28 @@ namespace fc::api::rpc {
       }
     }
     chans.clear();
+    reconnect(3, std::chrono::seconds(5));
   }
 
   void Client::_flush() {
-    if (!writing && !write_queue.empty()) {
+    if (!writing && !write_queue.empty() && not reconnecting) {
       auto &[id, buffer] = write_queue.front();
       writing = true;
-      socket.async_write(boost::asio::buffer(buffer.data(), buffer.size()),
-                         [=](auto &&ec, auto) {
-                           std::lock_guard lock{mutex};
-                           if (ec) {
-                             return _error(ec);
-                           }
-                           writing = false;
-                           write_queue.pop();
-                           _flush();
-                         });
+      socket->async_write(boost::asio::buffer(buffer.data(), buffer.size()),
+                          [=](auto &&ec, auto) {
+                            std::lock_guard lock{mutex};
+                            if (ec) {
+                              return _error(ec);
+                            }
+                            writing = false;
```

Comment:
move `writing = false` before `if (ec)` or add to `reconnect`, otherwise will never write anything after reconnect